tests/sles4sap/saptune/mr_test.pm

```diff
@@ -17,6 +17,19 @@ use version_utils 'is_sle';
 use strict;
 use warnings;
 
+sub reboot_wait {
+    my ($self) = @_;
+
+    $self->reboot;
+
+    # Wait for tuned to tune everything
+    if (is_sle('>=15')) {
+        assert_script_run "timeout 60 bash -c 'until tuned-adm verify >/dev/null ; do sleep 1 ; done'", 60;
```

Comment:
If I'm reading this correctly, this bash code can fail with the timeout if tuned is not fast enough, which would abort the test. Is that the intent?

Reply:
> If I'm reading this correctly, this bash code can fail with the timeout if tuned is not fast enough, which would abort the test. Is that the intent?

Yes.  If tuned takes more than one minute to configure the system, I would consider it a bug: bsc#1143291

We can add a variable but it wouldn't be useful as it would mask that bug.  Let's see if the developers care about making user sessions available only after tuned has finished configuring the system.